src/main/scala/apps/femur/IcpProposalRegistration.scala

```diff
@@ -53,16 +53,25 @@ object IcpProposalRegistration {
 
     val (model, modelLms, targetMesh, targetLms) = LoadTestData.modelAndTarget()
 
-    val numOfEvaluatorPoints = model.referenceMesh.pointSet.numberOfPoints // Used for the likelihood evaluator
-    val numOfICPPointSamples = numOfEvaluatorPoints // Used for the ICP proposal
-    val numOfSamples = 1000 // Length of Markov Chain
+    val numOfEvaluatorPoints = model.referenceMesh.pointSet.numberOfPoints/2 // Used for the likelihood evaluator
+    val numOfICPPointSamples = model.rank*2 // Used for the ICP proposal
+    val numOfSamples = 10000 // Length of Markov Chain
 
-    val proposalIcp = MixedProposalDistributions.mixedProposalICP(model, targetMesh, numOfICPPointSamples, projectionDirection = ModelAndTargetSampling)
+    // Closest Point proposal
+    // projectionDirection = TargetSampling (if registering partial meshes)
+    val proposalIcp = MixedProposalDistributions.mixedProposalICP(model, targetMesh, numOfICPPointSamples, projectionDirection = ModelAndTargetSampling, tangentialNoise = 100.0, noiseAlongNormal = 3.0, stepLength = 0.1)
+    // Random walk proposal
+//    val proposalRandom = MixedProposalDistributions.mixedProposalRandom(model)
```

Comment:
cleanup commented code in this file, or state why it should be there and when to uncomment the code